### ISSUE TITLE
Bug: Code that should be referencing input parameter `ha_azure_eip_name_resource_group` seems to use `azure_eip_name_resource_group`

### ISSUE THREAD

**Issue by gitet** [terraform-aviatrix-modules/terraform-aviatrix-mc-spoke#27]
When using `allocate_new_eip = false` and using PIPs already terraformed like this:

```
  allocate_new_eip                 = false # true  
  eip                              = azurerm_public_ip.pip-egress-spoke-gateway.ip_address
  azure_eip_name_resource_group    = "foo:rg-name"
  ha_eip                           = azurerm_public_ip.pip-egress-spoke-gateway-hagw.ip_address
  ha_azure_eip_name_resource_group = "bar:rg-name"
```
terraform plan wants to deploy this (relevant snippets shown):

```
      + azure_eip_name_resource_group         = "foo:rg-name"  # correct
      + eip                                   = "20.67.87.14"   # correct
      + ha_azure_eip_name_resource_group      = "foo:rg-name"  # WRONG
      + ha_eip                                = "20.67.87.225"  # correct
```

Could you please take a look at this?
Thanks!
/ET

**Comment by Dennizz** [comment]
Patched code in master branch. Will be part of next release.

**Comment by gitet** [comment]
I'll patch my local code in the meantime - thanks!

**Comment by Dennizz** [comment]
Patched in 1.6.8
